build/gulp.core.js

```diff
@@ -55,6 +56,36 @@ const createOutputDirs = () => {
     .pipe(gulp.dest('./out/bp/data/storage'))
 }
 
+const createMigration = cb => {
+  const args = require('yargs')(process.argv).argv
+  if (!args.ver) {
+    console.error('Version is required (set with --ver parameter')
+    console.error('Example: yarn cmd create:migration --target core --ver 13.0.0 --title "some config update"')
+    return cb()
+  }
+
+  const target = args.target || 'core'
+  const version = args.ver.replace(/[ .]/g, '_').replace('v', '')
+  const title = (args.title || '').replace(/[ .]/g, '_').toLowerCase()
+
+  const template =
+    target === 'core'
+      ? path.resolve(__dirname, '../src/bp/core/services/migration/template_core.ts')
+      : path.resolve(__dirname, '../src/bp/core/services/migration/template_module.ts')
+
+  const targetDir =
+    target === 'core'
```

Comment:
I know we didn't talk about this but did you think about how a 3rdParty module developer would create a migration for its module (being outside BP monorepo). I imagine he will simply copy paste a migration file in its root directory and see if it runs correctly when module is installed on a bp server

Reply:
Hmm I haven't thought about that one. Right now only the core version is used for migrations. It would add a bit of complexity to use each module's version for that purpose, but I think we could update it in the future without causing issues, what do you think? Plus, we would need to update the module version, which we don't do right now

Reply:
I think unofficial modules should have their own versions. For instance, they could release multiple versions of their module on the same version of Botpress. I think they might need their own separate migration process.